### ISSUE TITLE
average sentiment metric followup

### ISSUE THREAD

**Issue by mhow2** [crossminer/scava#329]
( This is about the project https://github.com/docdoku/docdoku-plm/issues in year 2018 )

Hi @creat89, @Danny2097, @yannisKork 

So as far as I understood from the mail thread about this metric, you committed the changes that makes the reader not to put value `0` when no message occur.

Now about this:
> Before that, there were no comments in 2018. That's why you don't have data before May.

I don't understand because I'm seeing many comments issued before May:

https://github.com/docdoku/docdoku-plm/issues/1141 there is a comment
on 27 Feb 2018

This one : https://github.com/docdoku/docdoku-plm/issues/1146 as one on
20 Apr 2018

https://github.com/docdoku/docdoku-plm/issues/1121 16 March 2018

**Comment by creat89** [comment]
Hello,

I'm guessing this issue is related to the pagination issue, which is preventing us to retrieve all the information.

**Comment by creat89** [comment]
Hello @mhow2,

I have pushed commit 9d02f6733daa72956d17b8b69370cddf1876d0af which should solve some of the issues that we were observing in the average sentiment and that were discussed in the email. Now, the calculation should only take into account the bugs that contain comments, instead of doing an average of all the bugs even if they don't contain comments. Also, I made some changes in a transient metric , as it was overwriting the values every time a new comment was added in a new day.
